esprima.js

```diff
@@ -4230,6 +4233,18 @@
                 }
 
                 lex();
+
+                if (match(')')) {
+                    possibleArrow = lookahead2();
+                    /* istanbul ignore else */
```

Comment:
TIL istanbul. Awesome idea that is.
I'm not familiar enough with the parser to come up with a test case that causes this condition to be false, hence this. If you can think of something, let me know and I'll update.